lib/Byng/Pimcore/Sitemap/Generator/SitemapGenerator.php

```diff
@@ -36,26 +36,30 @@
     /**
      * @var SimpleXMLElement
```

Comment:
What do you mean?

```
// Retrieve site trees
$config = simplexml_load_file(SITEMAP_CONFIGURATION_FILE);
$siteRoots = $config->sites->site;
```
=> The site parameter is a XML block containing several tags. I think `SimpleXMLElement` is the right type here, but I'm open to other alternatives